homeassistant/components/ebox/sensor.py

```diff
@@ -96,9 +156,10 @@ def __init__(self, ebox_data, sensor_type, name):
         """Initialize the sensor."""
         self.client_name = name
         self.type = sensor_type
-        self._name = SENSOR_TYPES[sensor_type][0]
-        self._unit_of_measurement = SENSOR_TYPES[sensor_type][1]
-        self._icon = SENSOR_TYPES[sensor_type][2]
+        metadata = SENSOR_TYPES[sensor_type]
+        self._name = metadata.name
```

Comment:
```suggestion
        self._attr_name = f"{self.client_name} {metadata.name}"
```

Reply:
Wont this cause renamed entities?

Reply:
Unless I'm missing something, it should be fine. That's the f-string from the `name` property.

Reply:
No, see the `name` property method (that can be removed because of this).

Reply:
Not sure about all these comments…however I have removed the @property name method and replaced it with self._attr_name, I believe that is the sum of the comments.